exampleCourse/questions/examplesMatrixOutput/question.html

```diff
@@ -0,0 +1,82 @@
+    <a href="https://prairielearn.readthedocs.io/en/latest/elements/#pl-matrix-output-element"><code>pl-matrix-output</code></a>.
+    The goal of <a href="https://prairielearn.readthedocs.io/en/latest/elements/#pl-matrix-output-element"><code>pl-matrix-output</code></a>
+    is to allow users to easily copy values for various computational tools.
+  </p>
+</pl-question-panel>
+
+<div class="card my-2">
+  <div class="card-body">
+    <pl-question-panel>
+      <p>Within this question, you can observe the default behavior of the <a href="https://prairielearn.readthedocs.io/en/latest/elements/#pl-matrix-output-element"><code>pl-matrix-output</code></a> element.
+        In particular, the default matrix output has a Matlab and a Python tabs and the parameters are displayed with <code>3</code> digits. This element has the ability to display any matrix or scalar value.</p>
+      </pl-question-panel>
+      <pl-matrix-output>
+        <variable params-name="a">a</variable>
+        <variable params-name="x">x</variable>
+        <variable params-name="x1">x1</variable>
+      </pl-matrix-output>
+    </div>
+  </div>
+
+  <div class="card my-2">
+    <div class="card-body">
+      <pl-question-panel>
+        <p>This example lowers the number of displayed digits to <code>0</code> and introduces comments for each variable. This uses the optional parameter <code>digits</code>.
+      </pl-question-panel>
+      <pl-matrix-output digits="0">
+        <variable params-name="x" params-comment="This is an array">x</variable>
+        <variable params-name="x1" params-comment="This is a 2D matrix">x1</variable>
+      </pl-matrix-output>
+    </div>
+  </div>
+
+  <div class="card my-2">
+    <div class="card-body">
+      <pl-question-panel>
+        <p>Under this variant, we opt to change the number of displayed digits to <code>0</code> for most variables but retain <code>2</code> displayed digits for other variables. This uses the optional variable parameter <code>params-digits</code>.</p>
+      </pl-question-panel>
+      <pl-matrix-output digits="0">
+        <variable params-name="a">a</variable>
+        <variable params-name="b">b</variable>
+        <variable params-name="c" params-digits="2">c</variable>
+        <variable params-name="xC" params-digits="2">xC</variable>
+      </pl-matrix-output>
+    </div>
+  </div>
+
+  <div class="card my-2">
+    <div class="card-body">
+      <pl-question-panel>
+        <p>Another option is to display a tab for Mathematica. This is very similar to Matlab syntax, but it takes into account Mathematica reserved variables (C, D, E, I, K, N, O) and formats comments correctly. This uses the optional parameter <code>display-mathematica-tab</code>.</p>
+      </pl-question-panel>
+      <pl-matrix-output digits="0" display-mathematica-tab="True">
+        <variable params-name="x" params-comment="The Mathematica tab adds an 'm' to the variable name">C</variable>
```

Comment:
When I run this, I don't see the `m` being added. I just see the plain variable names (like `C`).

Reply:
The `m` is only added in the Mathematica tab, I have clarified this in the example question explanation